src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+/* #region page */
+body {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
```

Comment:
Ensure that the path 'images/star-active.svg' is correct and that the file exists in the specified directory. This applies to all similar image paths in the file.